packages/lesswrong/server/dialogues/cron.ts

```diff
@@ -21,3 +24,81 @@ addCronJob({
     })
   }
 });
+
+const checkCompatible = (preferences:DbDialogueMatchPreference[]) => {
+  if (preferences.length >= 2) {
+
+    // Topic
+    const sourceUserYesTopics = getUserTopics(preferences[0]);
+    const targetUserYesTopics = getUserTopics(preferences[1]);
```

Comment:
I don't think there's any meaningful point-of-view from which one of these is the "sourceUser" and the other the "targetUser", since this is running in a cron job.  Naming is a bit confusing, given that